### ISSUE TITLE
Can't resolve hostname

### ISSUE THREAD

**Issue by philong013** [AlexProgrammerDE/balena-minecraft-server#25]
Hi uh I just balena push the package, everything is finish however when I tried to connect to the server using balenaminecraftserver it shows 'Can't Resolve hostname' and 'Unknown host'. I don't have any idea what went wrong there, you have any idea? Thanks in advance

**Comment by AlexProgrammerDE** [comment]
@philong013 does it show any errors in the dashboard?

**Comment by philong013** [comment]
@AlexProgrammerDE Error: Invalid or corrupt jarfile paper.jar? Is this the one
![er](https://user-images.githubusercontent.com/65332446/81903385-0caeb300-9605-11ea-8594-9a396b7bbd4d.PNG)

**Comment by AlexProgrammerDE** [comment]
@philong013 can you please set `ENABLE_UPDATE` to `true`?

**Comment by philong013** [comment]
@AlexProgrammerDE forgive my dumbness but how do i set it on the cmd?

**Comment by AlexProgrammerDE** [comment]
@philong013 you set it in the cloud dashboard. Look at this page, it describes how to do it: https://www.balena.io/docs/learn/manage/serv-vars/

**Comment by philong013** [comment]
Setting enable_update to true seems to improve however it still remain as 'Can't resolve hostname' and I got this error. 
![image](https://user-images.githubusercontent.com/65332446/81905246-ce66c300-9607-11ea-8bc6-4de72c45790e.png)

I'm connecting my raspberry Pi and Laptop to my phone's personal hotspot tho, is it an issue?
FYI I was setting up the server using my phone's hotspot but laptop was connecting to another wifi. Afterward I switched my laptop to personal hotspot after pushing the package, I don't know if its the main cause.
